python/cudf/cudf/tests/test_string.py

```diff
@@ -874,6 +874,25 @@ def test_string_contains(ps_gs, pat, regex, flags, flags_raise, na, na_raise):
         assert_eq(expect, got)
 
 
+@pytest.mark.parametrize(
+    "ptn,esc,expect",
+    [
+        ("abc", "", [True, False, False, False, False]),
+        ("b%", "/", [False, True, False, False, False]),
+        ("%b", ":", [False, True, False, False, False]),
+        ("%b%", "*", [True, True, False, False, False]),
```

Comment:
Hi, I'd like to clarify what does `esc` argument do here? My understand is `:` and `*` can be used as escape character, for example `:%` or `*%` can match literal `%`. The pattern doesn't contain either of the literal matching, should we add it?

Reply:
I just put these in to show they have no effect. I can add additional escape character tests.